packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -487,7 +487,10 @@ export class ProjectService {
                     JobStepType.COMPILING,
                     async () => {
                         try {
-                            return await adapter.compileAllExplores();
+                            return await adapter.compileAllExplores(
+                                false,
+                                this.lightdashConfig.warehouseTimezone,
```

Comment:
I guess in the future we could replace this lightdashCOnfig with a `project` setting after doing https://github.com/lightdash/lightdash/issues/5921